charts/flagsmith/templates/ingress-api.yaml

```
@@ -48,6 +49,7 @@ spec:
                 port: 
                   number: {{ $svcPort }}
             {{- else }}
+          {{- end }}
             backend:
               serviceName: {{ $fullName }}-api
               servicePort: {{ $svcPort }}
```

Had to move some lines around to make this work:

```suggestion
            {{- else }}
            backend:
              serviceName: {{ $fullName }}-api
              servicePort: {{ $svcPort }}
            {{- end }}
          {{- end }}
```